src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Color;
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random randomColor = new Random();
```

Comment:
random number generator has no common with color
```suggestion
    private final Random random = new Random();
```

Reply:
Ok, clear